scm/native/access.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2021 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package native
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-vela/types/library"
+	"github.com/jenkins-x/go-scm/scm"
+
+	"github.com/sirupsen/logrus"
+)
+
+// OrgAccess captures the user's access level for an org.
+func (c *client) OrgAccess(u *library.User, org string) (string, error) {
+	logrus.Tracef("Capturing %s access level to org %s", u.GetName(), org)
```

Comment:
Could we update the logging statements to reflect current state in the codebase? 👍 

i.e.

https://github.com/go-vela/server/blob/3fe3d7eaabc3f1ee03d3a7636749bdc8009ff0ee/scm/github/access.go#L16-L21

I think this change will need to be applied throughout the functions and files being added here.

Reply:
Not sure what you mean by "current state" in this context? Could you elaborate?

Reply:
Ohh, I'm sorry I see it now. You mean using the client logger. Sorry, for some reason my eyes weren't picking up the difference. Yeah, super easy change!

Reply:
Better to get it out of the way now tho. So good call out!